src/test/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/IndexStateManagementRestTestCase.kt

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.indexstatemanagement
+
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.IndexStateManagementPlugin.Companion.POLICY_BASE_URI
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.models.Policy
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.models.Policy.Companion.POLICY_TYPE
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.util._ID
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.util._VERSION
+import org.apache.http.HttpEntity
+import org.apache.http.HttpHeaders
+import org.apache.http.entity.ContentType
+import org.apache.http.entity.StringEntity
+import org.apache.http.message.BasicHeader
+import org.elasticsearch.client.Response
+import org.elasticsearch.common.settings.Settings
+import org.elasticsearch.common.unit.TimeValue
+import org.elasticsearch.common.xcontent.LoggingDeprecationHandler
+import org.elasticsearch.common.xcontent.NamedXContentRegistry
+import org.elasticsearch.common.xcontent.XContentParser.Token
+import org.elasticsearch.common.xcontent.XContentParserUtils.ensureExpectedToken
+import org.elasticsearch.common.xcontent.XContentType
+import org.elasticsearch.common.xcontent.json.JsonXContent
+import org.elasticsearch.rest.RestStatus
+import org.elasticsearch.test.ESTestCase
+import org.elasticsearch.test.rest.ESRestTestCase
+import org.junit.rules.DisableOnDebug
+
+abstract class IndexStateManagementRestTestCase : ESRestTestCase() {
+
+    private val isDebuggingTest = DisableOnDebug(null).isDebugging
+    private val isDebuggingRemoteCluster = System.getProperty("cluster.debug", "false")!!.toBoolean()
+
+    fun Response.asMap(): Map<String, Any> {
```

Comment:
NP: Function is small enough to be an expression, ex. ```fun Response.asMap(): Map<String, Any> = entityAsMap(this)```. Although [Kotlin recommends this](https://kotlinlang.org/docs/reference/coding-conventions.html#function-formatting) as long as its consistent with either one or the other it should be fine. I noticed you did it with ```XContentBuilder.string()``` in ```ElasticExtensions.kt```

Reply:
Fixed